assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -71,6 +75,24 @@ function MarkdownContent({ children }) {
   );
 }
 
+function LoadingFailure({ entity, error }) {
```

Comment:
Do we need this local components here?
The Loading and Notification boxes are generic, so if they need some improvement, I think we should do them here, to have them tested.
Because I see that we start adding terms like `entity` that nobody really understand in the long run.

Maybe we can convert the `text` prop of the `NotificationBox` in a list or so.

Reply:
Agreed. I'll take care of changing the `NotificationBox` as suggested.